features/install-cli.feature

```diff
@@ -52,6 +54,14 @@ Feature: Command behavior of client side tools installation
     Then the exit status should be 0
     And the binary "docker" should be installed
 
+    When I run `bundle exec vagrant service-manager install-cli docker --cli-version 1.12.1`
+    Then the exit status should be 0
+    And the binary "docker" of service "docker" should be installed with version "1.12.1"
```

Comment:
I think this step is also quite nice now. Allows us to do a dedicated check in its implementation